Artifacts/windows-seleniumgrid-hub/Artifactfile.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
+    "title": "Selenium-grid hub",
+    "description": "Use this artifact to setup your selenium-grid hub",
+    "publisher": "Microsoft",
+    "tags": [
+        "PowerShell",
+        "Selenium"
+    ],
+    "iconUri": "https://storagehsm6ciz.blob.core.windows.net/mycustomscripts/selenium-grid-logo.png",

```

Comment:
This isn't going to work for the long term.  Maybe just use the logo from selenium server: http://docs.seleniumhq.org/images/selenium-grid-logo.png